src/main/resources/application.conf

```diff
@@ -51,16 +58,20 @@ backend {
     // baseExecutionBucket = ""
     // endpointUrl = ""
 
+    // [Optional]
+    // If defined, data localization and delocalization will be done on behalf of another entity (typically a user).
+    // This allows cromwell to localize data that wouldn't be accessible
+    // using whatever "authScheme" has been defined in the "google" configuration (see below).
+    // It requires a user refresh token to be passed upon every workflow submission (see "workflow options" in cromwell documentation).
+    // client_id and client_secrets must be the ones that have been used to generate the above-mentioned refresh token.
+    // localizeWithRefreshToken = {
+    //   client_id = ""
+    //   client_secret = ""
+    // }
```

Comment:
It's a little unclear to me and I think I agree with Miguel about this belonging somewhere else.  I also don't fully understand the rest of the code yet